### PR TITLE
Add - Landing page layout version 1

### LINKED ISSUE #6: Create the landing page layouts - version 1
The landing page should contain the following sections:
- Manu,
- Banner,
- Category cards,
- Most Popular cards,
- Newsletter subscription,
- Social Icons,
- Footer

### PR DESCRIPTION
**Pull request by zaraayvazyan9**
The branch contains the 1st version of landing page layout which has:
Menu,
Banner,
Category cards,
"Most popular" cards,
Newsletter section,
Social Icons,
Footer

**NOTE: The stylings are not complete and are not final.**

### CODE REVIEW

File: about.html
Comment (MrTim) [inline about.html:29]: `myFunction` is not a proper name for function. Could you please come up with something more descriptive?)

File: css/normalize.css
Comment (MrTim) [inline css/normalize.css:1]: Why do you need this css file?

File: css/style.css
Comment (MrTim) [inline css/style.css:2]: Try to not leave commented code in the code you're submitting for review. Reason is - this code will probably live there forever and won't be removed (this is what happening in real life projects)

File: css/style.css
Comment (MrTim) [inline css/style.css:6]: But this comment is fine - because you're describing what code/styles below will relate to.

File: index.html
Comment (MrTim) [inline index.html:42]: `action="/action_page.php"` do you use it right now?

File: index.html
Comment (MrTim) [inline index.html:59]: question: is it ok to use these images? This is question more about future. Because if you want this site to be available in public => you should be sure which type of digital content you're using.

Comment (MrTim) [inline index.html:59]: Second comment to this line: as a next step - create an array of items you wan't to display on your site and do layout from javascript (because you already know how to create element and add it to your document).
But again - you can improve it as a next step.

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice progress! 👍  I left some comment, please adjust and we'll look at it again!

File: about.html
Comment (zaraayvazyan9) [reply about.html:29]: fair enough. I've changed it to "burgermenu".

File: css/normalize.css
Comment (zaraayvazyan9) [reply css/normalize.css:1]: In order to display my stylings in the same way on all the browsers I used this file. Wouldn't you recommend it?

File: css/style.css
Comment (zaraayvazyan9) [reply css/style.css:2]: Yes, sorry I forgot to delete it. Done. Thanks

File: index.html
Comment (zaraayvazyan9) [reply index.html:59]: Not sure I got you. It was a free image platform, with free download. Or maybe you mean the format?

File: index.html
Comment (zaraayvazyan9) [reply index.html:42]: True. I removed it for now.

**Comment by NadezhdaBabenko** [comment]
Please, make changes according to comments @MrTim

File: index.html
Comment (golovnevs) [reply index.html:59]: I think Ivan asked about the rights for the picture. If it is from a free image platform, it's ok.  My advice is to download images instead of using links to external resources because If the resource is not available or the picture is removed from it, picture will not be loaded on your page.

**Comment by zaraayvazyan9** [comment]
Hi guys,
I've applied some fixes in branch-1 and pushed them. 
Could you pls take a look? @MrTim @NadezhdaBabenko @golovnevs 
Thanks,
Zara

**Comment by NadezhdaBabenko** [comment]
Ok! I checked the commit and it looks good, but next time please wait for teacher's approve before merging a branch into the master;)

**Pull request merged**
